.github/scripts/defaults.sh

```diff
@@ -18,16 +18,22 @@ clean () {
 }
 
 # make parallelism
-CI_MAKE_NPROC=8
+CI_MAKE_NPROC=4
 # chosen based on a 24c system shared with 1 other project
 REMOTE_MAKE_NPROC=4
 
 # verilator version
 VERILATOR_VERSION=v4.034
 
 # remote variables
-REMOTE_PREFIX=$CI_DIR/$CIRCLE_PROJECT_REPONAME-$CIRCLE_BRANCH
-REMOTE_WORK_DIR=$REMOTE_PREFIX-$CIRCLE_SHA1-$CIRCLE_JOB
+
+CURRENT_BRANCH=$(git branch --show-current)
+
+# CI_DIR is defined externally based on the GH repository secret BUILDDIR
```

Comment:
Looks like `CI_DIR` isn't getting set (look at the `install-verilator` step trying to delete things that it shouldn't have access to because the path isn't fully specified). Can you verify that it is setup correctly (and make sure that no `rm`s are done without it setup)?